src/neuronmusic/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+# a lot of this copied from nrn/share/lib/python/neuron/rxd/geometry3d/CMakeLists.txt
+
+# ~~~
+# Following set in src/nrnpython/CMakeLists.txt should be made global.
+# These are used in setup.py.
+# ~~~
+set(NRN_SRCDIR ${PROJECT_SOURCE_DIR})
+set(CC ${CMAKE_C_COMPILER})
+set(CXX ${CMAKE_CXX_COMPILER})
+
+nrn_configure_file(setup.py src/neuronmusic)
+
+if(NRN_ENABLE_MUSIC
+   AND NRN_ENABLE_MODULE_INSTALL
+   AND NRN_ENABLE_PYTHON)
+
+  # Prepare a shell script to run python setup.py
+  file(
+    WRITE ${CMAKE_CURRENT_BINARY_DIR}/runpy.sh
+    "\
+#!bash\n\
```

Comment:
I wouldn't complain here as I didn't check #1896 earlier. But I think we can say there is room for improving this...Just a side note!